integration-tests/features/device-api.feature

```diff
@@ -202,6 +202,55 @@ Feature: Device API
       }
       """
 
+    # Bob can update if device is no more behind symmetric NAT.
+    When I PATCH path "/api/devices/${device_id}" with json body:
+      """
+      {
+        "symmetric_nat": false
```

Comment:
In my testing, this was not present in the request at all

Reply:
Yeah.. this PR changes AddDevice model, but that's not used in the PATCH operation.